scripts/postinstall.js

```diff
@@ -8,26 +8,41 @@ const isWindows = process.platform === 'win32';
 
 const truthyStr = (val) => val && !['0', 'false', 'f', 'n', 'no'].includes(val.toLowerCase());
 const { CI, ADBLOCK, SILENT } = process.env;
-if (!truthyStr(CI) && !truthyStr(ADBLOCK) && !truthyStr(SILENT)) {
-  const messageTokens = ['Serverless Framework successfully installed!'];
-  if (isStandaloneExecutable && !isWindows) {
-    messageTokens.push(
-      'To start your first project, please open another terminal and run “serverless”.',
-      'You can uninstall at anytime by running “serverless uninstall”.'
+const isNpmGlobalPackage = require('../lib/utils/npmPackage/isGlobal');
+const isNpmPackageWritable = require('../lib/utils/npmPackage/isWritable');
+
+(async () => {
+  if (!truthyStr(CI) && !truthyStr(ADBLOCK) && !truthyStr(SILENT)) {
+    const messageTokens = ['Serverless Framework successfully installed!'];
+
+    if (
+      (isStandaloneExecutable && !isWindows) ||
+      ((await isNpmGlobalPackage()) && (await isNpmPackageWritable()))
```

Comment:
I believe there's no point to do `isNpmPackageWritable` check, as we're in context of a shell which just written new installation (so we can assume it's true)

Reply:
Good point, I'm going to remove it :+1: